### PR TITLE
Refactors options page to use choo

### PR DESCRIPTION
**Pull request by alanshaw**
\o/ nothing changes visually, but options page now uses choo!...bringing it in line with the other pages

* Modules for each form/section
* Simplifies the code
* Removes `lib/data-i18n.js` as it's no longer used anymore

### CODE REVIEW

**Comment by lidel** [comment]
@alanshaw  I see some files related to dir listing. 
This should be merged after #323, right?

File: add-on/src/options/forms/api-form.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+'use strict'
+/* eslint-env browser, webextensions */
+
+const browser = require('webextension-polyfill')
+const html = require('choo/html')
+
+function apiForm ({ ipfsApiUrl, ipfsApiPollMs, automaticMode, onOptionChange }) {
+  const onIpfsApiUrlChange = onOptionChange('ipfsApiUrl')
```
Comment (lidel) [inline add-on/src/options/forms/api-form.js]: This should be normalized just like `customGatewayUrl` is

**Comment by alanshaw** [comment]
@lidel closing as I've opened #337 which has only the options page changes in and the change requested above (it doesn't depend on #323).

I'm not Android enabled so unfortunately I cannot verify, sorry :(

**Comment by lidel** [comment]
@alanshaw No problem, I'll test  #337 on Android and let you know if any issues.

**Pull request closed without merging**